### PR TITLE
feature/864_repo_refactoring_multi_agent___cygnus_common_doc

### PR DESCRIPTION
**Pull request by frbattid**
* Partially implements issue #864 
* No tests are required
* Assignee @pcoello25

### CODE REVIEW

File: doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Content:
+
+* [Introduction](#section1)
+* [`cygnus_instance_<id>.conf`](#section2)
+* [`agent_<id>.conf`](#section3)
+
+##<a name="section1"></a>Introduction
+Any Cygnus agent is configured through two different files:
+
+* A `cygnus_instance_<id>.conf` file addressing all those non Flume parameters, such as the Flume agent name, the specific log file for this instance, the administration port, etc. This configuration file is not necessary if Cygnus is run as a standlalone application (see later), but it is mandatory if run as a service (see later).
```
Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md]: standlalone -> standalone

File: doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* An `agent_<id>.conf` file addressing all those Flume parameters, i.e. how to configure the different sources, channels, sinks, etc. that compose the Flume agent behind the Cygnus instance. always mandatory.
```
Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md]: always -> Always ?

File: doc/cygnus-common/installation_and_administration_guide/introduction.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+##Intended audience
+This document is mainly addressed to those FIWARE users willing to create historical views about any source of data handled by any of the available Cygnus agents. In that case, you will need this document in order to learn how to install and administrate cygnus-common.
+
+If your aim is to contribute to cygnus-common, please refer to the [User and Programmer Guide](../user_and_programmer_guide/introduction.md).
+
+[Top](#top)
+
+##Structure of the document
+Apart from this introduction, this Installation and Administration Guide mainly contains sections about installing, configuring, running and testing cygnus-common. The FIWARE user will also find useful information regarding logs and alarmas, how to manage a Cygnus agent through the RESTful interface and important performance tips. In addition, sanity check procedures (useful to know wether the installation was successful or not) and diagnosis procedures (a set of tips aiming to help when an issue arises) are provided as well.
```
Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/introduction.md]: alarmas -> alarms

**Comment by pcoello25** [comment]
LGTM

File: doc/cygnus-common/installation_and_administration_guide/introduction.md (hunk 3 above)
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/introduction.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/981/commits/76eec6e2ee9eecc6a9fd83458649811f14bcb89b

File: doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md (hunk 2 above)
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/981/commits/76eec6e2ee9eecc6a9fd83458649811f14bcb89b

File: doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md (hunk 1 above)
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/981/commits/76eec6e2ee9eecc6a9fd83458649811f14bcb89b

**Pull request merged**